ui/client/containers/ProcessesList.tsx

```diff
@@ -80,6 +80,8 @@ function useFiltersState(defaultQuery: Queries) {
   return {search, filters, setFilters}
 }
 
+const sortFunction = (a: string, b: string) => a.toLowerCase().localeCompare(b.toLowerCase())
```

Comment:
Shouldn't it be simply a cached instance of  `Intl.Collator().compare`?

```javascript
> ["a", "A", "Z", "z", "1", "二", "一"].sort(Intl.Collator().compare)
[ "1", "a", "A", "z", "Z", "一", "二" ]
```

Reply:
sure, corrected